app.js

```diff
@@ -21,8 +23,12 @@ const questionsRoutes = require("./routes/questionRoutes.js");
 const authMiddleware = require("./middleware/authMiddleware");
 // questions routes middleware??
 app.use("/api/questions", authMiddleware, questionsRoutes);
+//answer route
+const answerroutes = require("./routes/getanswerroute.js");
 
-// answers routes middleware ??
+// answers routes middleware ?
+//answer route midilware
+app.use("/api/answer", authMiddleware, answerroutes);
 async function start() {
   try {
     const result = await dbConnection.execute('select "test"');
```

Comment:
no need of creating variable here.
just put this only -> ` await dbConnection.execute('select "test"');`

Reply:
Thanks miki. this one on the main branch from Gizish. I only add here answer endpoint api

Reply:
done